soap/client.go

```diff
@@ -11,10 +11,9 @@ import (
 	"reflect"
 )
 
+// XSINamespace is a constant describing the standard namespace for XML schema
 const XSINamespace = "http://www.w3.org/2001/XMLSchema-instance"
 
-var XMLTyperType reflect.Type = reflect.TypeOf((*XMLTyper)(nil)).Elem()
```

Comment:
While fixing the issue, found this global variable.  No real reason this should be a global variable -  if we need to return the same element, provide it as a function.